client/my-sites/theme/main.jsx

```diff
@@ -626,11 +628,19 @@ class ThemeSheet extends React.Component {
 
 		const plansUrl = siteSlug ? `/plans/${ siteSlug }/?plan=value_bundle` : '/plans';
 
-		const { canonicalUrl, currentUserId, description, name: themeName } = this.props;
+		const {
+			canonicalUrl,
+			currentUserId,
+			description,
+			name: themeName,
+			seoTitle,
+			seoDescription,
+		} = this.props;
+
 		const title =
-			themeName &&
+			( seoTitle || themeName ) &&
 			i18n.translate( '%(themeName)s Theme', {
```

Comment:
If `seoTitle` is set would it make sense to just use that unchanged, instead of adding "Theme" at the end with the `%(themeName)s Theme` format?

Reply:
@travisw Good idea. Thank you! I will update shortly.